src/cljs/main/broadfcui/page/workspace/monitor/submission_details.cljs

```diff
@@ -136,7 +137,9 @@
   {:render
    (fn [{:keys [state props this]}]
      (let [server-response (:server-response @state)
+           nav-context (:nav-context props)
            {:keys [submission error-message]} server-response]
+       (utils/cljslog nav-context)
```

Comment:
Please remove log statements prior to merge (go ahead and create a separate PR and I'll review).